### PR TITLE
Add filters

### PR DESCRIPTION
**Pull request by jeetbhalani15**
## Added Filter Functionality
- Added Sort By Priority Filters.
- Added  Sort By Date Filters.
- And Added Sort by Tags.
 
Using hardcore Colors will change it soon.

### CODE REVIEW

File: Gravity-Notes/src/Components/Archive-card/ArchiveCard.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ export const ArchiveCard = ({ archive })=>{
         <div className="title-box">
         <h1 className="title-txt">{archive.title}</h1>
         <small>{archive.data}</small>
+        <span className={`priority-tag-${archive.priority} && ${archive.priority}`}>{archive.priority}</span>
         </div>
         <p dangerouslySetInnerHTML={{ __html: archive.content}}/>
```
Comment (Aditi0505) [inline Gravity-Notes/src/Components/Archive-card/ArchiveCard.js:25]: What is the purpose of using this? 
Not sure if it is a good practice. Have you come across an alternate?

File: Gravity-Notes/src/Components/Filters/Filter.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    align-items: center;
+    justify-content: flex-start;
+    gap: 7.5rem;
+    margin-bottom: .5rem;
+    cursor: default;
+}
+.filter-sub-heading{
+    font-size: 1.5rem;
+    color: #ff8e4c;
+    background-color: aliceblue;
```
Comment (Aditi0505) [inline Gravity-Notes/src/Components/Filters/Filter.css:24]: Please change this.

File: Gravity-Notes/src/Components/Notes-card/Notecard.js [hunk 3]
```diff
@@ -68,14 +77,24 @@ export const Notecard = ({ note }) => {
     console.log("add")
     console.log(currentAddedTag)
```
Comment (Aditi0505) [inline Gravity-Notes/src/Components/Notes-card/Notecard.js:78]: Please remove stray console logs

File: Gravity-Notes/src/Components/TextArea.js/Textarea.js [hunk 4]
```diff
@@ -108,6 +117,8 @@ export const Textarea = () => {
             onClick={handleExpand}
             required
             > </textarea> */}
```
Comment (Aditi0505) [inline Gravity-Notes/src/Components/TextArea.js/Textarea.js:119]: Remove commented code if not needed.

File: Gravity-Notes/src/Utils/DeleteNote.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import axios from "axios"
 
-  export const deleteNote = async (item,authState,noteDispatch, setNote)=>{
+  export const deleteNote = async (item,authState,noteDispatch, setNote, setFilter)=>{
     console.log("delete")
     console.log(authState.token)
     console.log(item._id)
```
Comment (Aditi0505) [inline Gravity-Notes/src/Utils/DeleteNote.js:6]: Remove stray console logs

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions for you to look at. 
Rest looks good!

**Review by anupkgurung — APPROVED** [review]
suggestion : do format the code specially the logical part instead of a one liner.
Rest looks good.
separation of logic is nicely used

**Pull request merged**